manifests/ext-auth-securitypolicy.yaml

```diff
@@ -62,9 +62,10 @@ spec:
       name: ext-auth-poc
   extAuth:
     recomputeRoute: true
-    grpc:
+    http:
+      headersToBackend: ["x-current-user"]
       backendRefs:
-        - name: grpc-ext-auth
+        - name: ext-auth-svc
           kind: Backend
           group: gateway.envoyproxy.io
           port: 9002
```

Comment:
Port is not required here. It can be removed after https://github.com/envoyproxy/gateway/issues/4999 is fixed.